examples/01_brain_alignment/plot_2_aligning_brain_sparse.py

```diff
@@ -685,3 +685,53 @@ def plot_surface_map(
 )
 
 plt.show()
+
+# %%
+# We can also partially transport feature maps, by specifying
+# a parameter ``id_reg`` which balances the amount of regularization
+# applied between the tarnsport plan and the identity mapping. The
+# regularized transport plan is then ``id_reg`` times the identity
+# plus ``(1 - id_reg)`` times the transport plan computed by FUGW.
+
+# Set of weights for the regularization parameter
+id_reg = 1 - np.linspace(0, 1, 3)
```

Comment:
I'd be in favour of writing this explicitly like so:
```suggestion
id_reg = [.0, .5, 1.]
```